xarray/core/common.py

```diff
@@ -343,17 +343,21 @@ def groupby(self, group, squeeze=True):
             group = self[group]
         return self.groupby_cls(self, group, squeeze=squeeze)
 
-    def rolling(self, min_periods=None, center=False, **kwarg):
-        """Returns a Rolling object for performing moving window operations.
+    def rolling(self, min_periods=None, center=False, **windows):
+        """
+        Moving window object.
 
         Parameters
         ----------
+        obj : Dataset or DataArray
```

Comment:
rrrrrrrr... I didn't mean to copy this line. This is needed in the `Rolling.__init__` docstring so that's how it got here.  I'll issue another PR to fix this.